### PR TITLE
9/make test app actually work

### PR DESCRIPTION
**Pull request by alex-eg**

### CODE REVIEW

File: a/unittests/lbge-unittests.asd [hunk 1]
```diff
@@ -0,0 +1,9 @@
+(defpackage :lbge-unittests-system
+  (:use :cl))
+
+(in-package :lbge-unittests-system)
+
+(asdf:defsystem :lbge-unittests
```
Comment (vaartis) [inline a/unittests/lbge-unittests.asd]: IDK about `unittests` everywhere, maybe `unit-tests`?

File: engine/math/t/vector.lisp [hunk 2]
```diff
@@ -1,149 +1,151 @@
...
 
+(in-package :lbge.test.vector)
 
 (deftest add-test
   (testing "2d vector addition"
-	   (ok (eq (add (make-float2 1 2)
-			(make-float2 3 4))
-		   (make-float2 4 6))))
+    (ok (eq (add (make-float2 1 2)
+		 (make-float2 3 4))
```
Comment (vaartis) [inline engine/math/t/vector.lisp]: What's with the tabs & spaces here..

File: a/unittests/lbge-unittests.asd (hunk 1 above)
Comment (alex-eg) [reply a/unittests/lbge-unittests.asd]: May be `unit-tests` as well.

File: engine/math/t/vector.lisp (hunk 2 above)
Comment (alex-eg) [reply engine/math/t/vector.lisp]: Oh. It's a fuckup.

File: a/unit-tests/lbge-unittests.asd [hunk 3]
```diff
@@ -0,0 +1,9 @@
+(defpackage :lbge-unit-tests-system
```
Comment (Lamppilutti) [inline a/unit-tests/lbge-unittests.asd]: After changes from _unittests_ to _unit-tests_ this file still has _unittests_ in its name.

Comment (alex-eg) [reply a/unit-tests/lbge-unittests.asd]: Really. Knew I've missed something.

**Pull request merged**